prod/westeurope/internal/api/functions_bonus/function_app/terragrunt.hcl

```diff
@@ -74,6 +74,7 @@ inputs = {
     COSMOSDB_BONUS_URI           = dependency.cosmosdb_bonus_account.outputs.endpoint
     COSMOSDB_BONUS_KEY           = dependency.cosmosdb_bonus_account.outputs.primary_master_key
     COSMOSDB_BONUS_DATABASE_NAME = dependency.cosmosdb_bonus_database.outputs.name
+    COSMOSDB_CONNECTION_STRING   = "AccountEndpoint=${dependency.cosmosdb_bonus_account.outputs.endpoint};AccountKey=${dependency.cosmosdb_bonus_account.outputs.primary_master_key};"
```

Comment:
you can also use the join function:
https://www.terraform.io/docs/configuration/functions/join.html

Reply:
can you change it accordingly? :|

Reply:
@gunzip I do

Reply:
The join function is usefull when you need to concatenate using a "separator". In this case interpolation is better.

Reply:
ok

Reply:
If you want you can add the connection_string as output to the cosmosdb module and than use it here.

Reply:
ok  I'll do that

Reply:
version 2.0.30 of the module is now required